nmostesting/NMOSTesting.py

```diff
@@ -1115,8 +1115,8 @@ def main(args):
         repo = git.Repo(".")
         TOOL_VERSION = repo.git.rev_parse(repo.head.object.hexsha, short=7)

```

Comment:
This format is literally just the bare git short hash... maybe we need to make this consistent with the `<owner>/<repo>[:<branch>]@<commit>` form?